### DISCUSSION TITLE
I'm getting this error when trying to connect instagram account.

### DISCUSSION THREAD

**Discussion by hmseeb** [pocketbase/pocketbase#6181] [Q&A] [answered]
flutter: ClientException: {url: https://connectobia.pockethost.io/api/collections/influencers/auth-with-oauth2, isAbort: false, statusCode: 400, response: {data: {password: {code: validation_invalid_format, message: Invalid value format.}}, message: Failed to create record., status: 400}, originalError: null}
Reloaded 1 of 3120 libraries in 728ms (compile: 77 ms, reload: 221 ms, reassemble: 198 ms).

**Comment by ganigeorgiev** [comment]
I remember you from https://github.com/pocketbase/pocketbase/discussions/6048 (_please do followup to your questions_).

If you have custom "Validation pattern" (_or min/max limits_) set for the `password` field, the builtin OAuth2 account random generated password may fail.

I'll consider excluding it with the next release from the OAuth2 validators (_Done_) but for now you can remove the extra field validations OR submit your own random generated password that matches your validation pattern with the OAuth2 call as part of the `createData` param (see https://pub.dev/documentation/pocketbase/latest/pocketbase/RecordService/authWithOAuth2.html).

**Reply by hmseeb** [reply]
So sorry about that.. I'm new to discussions. I removed it now I'm getting failed to fetch OAuth2 token

**Reply by ganigeorgiev** [reply, marked as answer]
I'm not sure what could be the reason for the latter error. You can try to start the executable with `--dev` and/or check the logs in the _Dashboard > Logs_ for more details.

If you are still not able to find the culprit, feel free to share a **minimal** reproducible and I'll try to investigate it (_~note that if you are testing with Flutter Web be aware that the streamed responses, used for the realtime events and the OAuth2 all-in-one call, are not supported out of the box - https://github.com/pocketbase/dart-sdk?tab=readme-ov-file#limitations~ this actually may no longer be true as recently https://github.com/dart-lang/http/pull/1401 got merged; I'll test it with the next release as it may require some adjustments from our end but in all cases, if it doesn't work with Flutter Web, check the [Limitations](https://github.com/pocketbase/dart-sdk?tab=readme-ov-file#limitations) section for a workaround_).

**Reply by hmseeb** [reply]
Fixed it! Thank you so much for you help... You're a genius
